### DISCUSSION TITLE
Request to add download button

### DISCUSSION THREAD

**Discussion by Aruraka** [PyPiSan/KinAni#21] [Ideas]
Hii developers if this app ..pls can u guys add a section for watching chinese dramas also? I think it would be great if u would introduce an extension called kisskh cos it have all dramas ..and can u guys please make the videos available there to download in the app itself too instead of in internal memory of phone please? It's a request..Hoping to hear from u guys

**Comment by PyPiSan** [comment]
Hi, Chinese drama is already there, I will add a search page for drama so you can search it. Downloading or offline watching is not possible currently.

**Comment by Aruraka** [comment]
Hii can u please tell when will u add download button ?

**Reply by PyPiSan** [reply]
It is not possible. It has streaming media, it can't be downloaded.

**Reply by Aruraka** [reply]
Ohh I see
